src/20_Components/amp-form.html

```diff
@@ -0,0 +1,78 @@
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+    Import the library.
+  -->
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script> 
+  <!-- 
+    Customize success or error messages by using `amp-form-submit-success` and `amp-form-submit-error` classes.
+   -->
+  <style amp-custom>
+        form.amp-form-submit-success [submit-success],  
+          form.amp-form-submit-error [submit-error]{
+            margin-top: 4px;
+            font-size: 20px;
+        }
+        form.amp-form-submit-success [submit-success] {
+            color: #ff4081;
+        }
+        form.amp-form-submit-error [submit-error] {
+            color: #dc4e41;
+        }
+        .button-submit {
+          margin-top: 4px;
+        }
+    </style>
+</head>
+<body> 
+
+  <!-- #### HTTP Request -->
+  <!--
+    Use the `action` attribute to provide the URL for the form action. The URL must be HTTPS. The subscribe button opens a new page.
+  -->
+  <form method="post" action="/submitForm" target="_blank">   
+      <input type="text" name="name" placeholder="Name..." required>
+      <input type="email" name="email" placeholder="Email..." required>
+      <input type="submit" value="Subscribe" class="button button-primary button-submit">
+  </form>
+
+  <!-- #### XHR Request -->
+  <!--
+    Provide `action-xhr` to send a XMLHttpRequest (XHR) together with a template, like [amp-mustache](https://github.com/ampproject/amphtml/blob/master/extensions/amp-mustache/amp-mustache.md); `amp-form` displays the `submit-success` or `submit-error` elements based on the response and renders the template data inside these 2 elements. 
```

Comment:
sugg: "Use  the `action-xhr` attribute to submit the form via XMLHttpRequest (XHR). You can use [amp-mustache](https://github.com/ampproject/amphtml/blob/master/extensions/amp-mustache/amp-mustache.md) templates to show custom success or error messages. . The `amp-form` component displays `submit-success` or `submit-error` elements based on the response and renders the template data inside these two elements. "